.github/workflows/pr-go.yaml

```diff
@@ -152,6 +160,6 @@ jobs:
           go-version: ${{ env.GO_VERSION }}
           cache: true
       - name: Install golangci-lint
-        run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@${{ env.GOLANGCI_LINT }}
+        run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
```

Comment:
Is there a reason to use the latest version?
This could introduce instabilities to the CI.

Reply:
I set latest here as goimport.
https://github.com/bucketeer-io/bucketeer/blob/c2a78137cf9bdbcd8dd5b01442b7e21b2a622d39/.github/workflows/pr-go.yaml#L65

Also, golangci-lint does not affect codes while mockgen and protoc do.

Reply:
I asked because last year, an older version broke the CI due to new linters, deprecation, etc.